erc20-load-test-tool/src/main/java/com/vmware/ethereum/config/AppConfig.java

```diff
@@ -93,6 +97,24 @@ private OkHttpClient okHttpClient(String correlationPrefix) throws GeneralSecuri
         .build();
   }
 
+  @Bean
+  public ArrayList<Credentials> credentialsArray(WorkloadConfig config) {
```

Comment:
Now that you have beans to create multiple credentials and sender addresses, you might want to clean up the beans related to single credentials and address.

Reply:
I am using different creds for deployer and senders, deployer -> deploys smart contract and distribute tokens.
so the single creds are for deployer, maybe I can change the name to make it clear